src/app/clients/ClientsPage.tsx

```diff
@@ -0,0 +1,195 @@
+"use client";
+
+import React, { Suspense, useState } from "react";
+import Table, { Row, TableHeaders } from "@/components/Tables/Table";
+
+import styled, { useTheme } from "styled-components";
+import { ClientsTableRow } from "@/app/clients/getClientsTableData";
+import FlaggedForAttentionIcon from "@/components/Icons/FlaggedForAttentionIcon";
+import PhoneIcon from "@/components/Icons/PhoneIcon";
+import CongestionChargeAppliesIcon from "@/components/Icons/CongestionChargeAppliesIcon";
+import DeliveryIcon from "@/components/Icons/DeliveryIcon";
+import CollectionIcon from "@/components/Icons/CollectionIcon";
+import ExpandedClientDetails from "@/app/clients/ExpandedClientDetails";
+import ExpandedClientDetailsFallback from "@/app/clients/ExpandedClientDetailsFallback";
+import Icon from "@/components/Icons/Icon";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
+import Modal from "@/components/Modal/Modal";
+import { Schema } from "@/supabase";
+import { Paper } from "@mui/material";
+
+const ClientsTablePaper = styled(Paper)`
+    margin: 1rem;
+    padding: 1rem;
+    border-radius: 1rem;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
+`;
+
+const collectionCentreToAbbreviation = (
+    collectionCentre: Schema["parcels"]["collection_centre"]
+): string => {
+    switch (collectionCentre) {
+        case "Brixton Hill - Methodist Church":
+            return "BH-MC";
+        case "Clapham - St Stephens Church":
+            return "CLP-SC";
+        case "N&B - Emmanuel Church":
+            return "NAB-EC";
+        case "Streatham - Immanuel & St Andrew":
+            return "STM-IS";
+        case "Vauxhall Hope Church":
+            return "VHC";
+        case "Waterloo - Oasis":
+            return "WAT-OA";
+        case "Waterloo - St George the Martyr":
+            return "WAT-SG";
+        case "Waterloo - St Johns":
+            return "WAT-SJ";
+        default:
+            return "-";
+    }
+};
+
+export const clientTableHeaderKeysAndLabels: TableHeaders = [
+    ["iconsColumn", ""],
+    ["fullName", "Name"],
+    ["familyCategory", "Family"],
+    ["addressPostcode", "Postcode"],
+    ["deliveryCollection", ""],
+    ["packingDate", "Packing Date"],
+    ["packingTimeLabel", "Time"],
+    ["lastStatus", "Last Status"],
+];
+
+const toggleableHeaders = [
+    "fullName",
+    "familyCategory",
+    "addressPostcode",
+    "packingDate",
+    "packingTimeLabel",
+    "lastStatus",
+];
+
+const clientTableColumnStyleOptions = {
+    iconsColumn: {
+        grow: 0,
+    },
```

Comment:
I think the iconColumns doesn't work if both icons are shown the same time (as in the second logo will overlap with the name column) - probably need some fix

Reply:
Yes, that's true. I've fixed it now by setting a `minWidth`